### ISSUE TITLE
433MHz Arduino Transmission Code for HomeWizard?

### ISSUE THREAD

**Issue by EWdG** [ghoss/HomeWizard#1]
Would the programmer of this library also have the option and would he take the time to write a transmission code for the Arduino?

**Comment by gh055** [comment]
I might be able to look into it, especially since I plan to procur another one of those transmitter units for a different project anyway. However, what is the use case if I may ask? You do realize that this would merely emulate the function of a standard HomeWizard on/off switch?

**Comment by EWdG** [comment]
With your library "HomeWizard" I can record the data from a transmitter to an actuator.
Now I want to send the recorded data from an Arduino to the actuator. For this a library with which one can send the recorded data is missing.
In the end, this Arduino would be integrated into the WLAN. There he would be able to operate with Virtuino (http://virtuino.com) with a smartphone or tablet.

**Comment by EWdG** [comment]
To include your HomeWizard library.
The download of the ZIP-file "HomeWizard-master.zip" could not simply be entered into the Arduiono IDE. I always got the message "Specified Folder / ZIP file does not contain a valid library".
Only when I unzipped the "HomeWizard-master.zip" by hand, I could use the then obtained "HomeWizard.zip" in the IDE.
I also had to separately download and integrate the "RFControl-master.zip". Especially since an empty directory "RFControl" can be found under "HomeWizard-master.zip" I wonder if this is what you want?
Please also see:
https://www.arduinoforum.de/arduino-Thread-Angegebener-Ordner-ZIP-Datei-enth%C3%A4lt-keine-g%C3%BCltige-Bibliothek

**Comment by EWdG** [comment]
Sorry. Wrong button pressed

**Comment by gh055** [comment]
You're right about the ZIP file. I originally added `RFControl` as a _submodule_, one major disadvantage of which is that GitHub does not include the submodule's code in the ZIP files. In the meantime, this method has become obsolete with the introduction of _subtrees_, which fixes this issue.

I've modified the repository and `RFControl`is now a subtree, so ZIP downloads will include the source code of that library as well. I just tested it with my IDE, and everything compiles ok from `HomeWizard-master.zip`. If it still doesn't work for you, please let me know.

**Comment by gh055** [comment]
>Now I want to send the recorded data from an Arduino to the actuator. For this a library with which one can send the recorded data is missing.

Ok, I understand now. It should not be too much trouble to add a transmission function. However, I still need to wait for a 433MHz module for testing to arrive, so it will take a week or two until I can start looking into this.

**Comment by EWdG** [comment]
I'll wait. Thank you in advance.
